### DISCUSSION TITLE
Weekly Lineup and Scoring data - Season before 2019

### DISCUSSION THREAD

**Discussion by jberkowitz89** [cwendt94/espn-api#145] [Q&A] [answered]
I've been using the package over the last few days, and I've been having some trouble pulling weekly player scoring data and weekly lineup scoring data for my league for seasons before 2019. 

I have a suspicion that this is a limitation from ESPN's end, since they don't seem to store box score data before 2019.  However, I wanted to confirm that this is the case. 

So, has anyone else been able to pull weekly player scoring and weekly lineup data for seasons before 2019?

**Comment by cwendt94** [comment, marked as answer]
Correct, any season before 2019 ESPN only shows the scores between teams every week and each teams final roster. If you want to get players weekly stats up to 2017 check out this [discussion](https://github.com/cwendt94/espn-api/discussions/131)
